src/util/encoding.ts

```diff
@@ -43,3 +43,7 @@ export function bufferToBigInt(bytes: Buffer | Uint8Array): bigint {
     bytes = Buffer.from(bytes)
     return BigInt("0x" + bytes.toString("hex"))
 }
+
+export function strip0x(str: string): string {
```

Comment:
Isn't this repeated? 
~~This had already been added in a previous PR~~  There is an approved PR that introduces `ensure0x` and `strip0x`... but it is not yet merged because a core dependency has an unsolved bug 🤷